libs/server-sdk/src/data_sources/data_source_status_manager.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <functional>
+#include <mutex>
+
+#include <boost/signals2.hpp>
+
+#include <launchdarkly/connection.hpp>
+#include <launchdarkly/data_sources/data_source_status_manager_base.hpp>
+#include <launchdarkly/server_side/data_source_status.hpp>
+
+namespace launchdarkly::server_side::data_sources {
+
+class DataSourceStatusManager
+    : public internal::data_sources::DataSourceStatusManagerBase<
```

Comment:
Make a data source status manager for X type of status which inherits from Y interface.